source/Engine.cpp

```diff
@@ -239,6 +241,35 @@ namespace {
 
 	const double RADAR_SCALE = .025;
 	const double MAX_FUEL_DISPLAY = 5000.;
+
+	const double CAMERA_VELOCITY_TRACKING = 0.1;
+	const double CAMERA_POSITION_CENTERING = 0.01;
+
+	pair<Point, Point> NewCenter(const Point &oldCenter, const Point &oldCenterVelocity,
+		const Point &baseCenter, const Point &baseVelocity)
+	{
+		if(Preferences::CameraAcceleration() == Preferences::CameraAccel::OFF)
+			return make_pair(baseCenter, baseVelocity);
+
+		double cameraAccelMultiplier = Preferences::CameraAcceleration() == Preferences::CameraAccel::REVERSED ? -1. : 1.;
+
+		const auto lerp = [](const auto &a, const auto &b, const auto &c)
+		{
+			return a + (b - a) * c;
+		};
+
+		// Flip the velocity offset if cameraAccelMultiplier is negative to simplify logic.
+		const auto absoluteOldCenterVelocity = lerp(baseVelocity, oldCenterVelocity, cameraAccelMultiplier);
+
+		const auto newAbsVelocity = lerp(absoluteOldCenterVelocity, baseVelocity, CAMERA_VELOCITY_TRACKING);
+
+		const auto newCenter = lerp(oldCenter + newAbsVelocity, baseCenter, CAMERA_POSITION_CENTERING);
+
+		// Flip the velocity back over the baseVelocity
+		const auto newVelocity = lerp(baseVelocity, newAbsVelocity, cameraAccelMultiplier);
```

Comment:
```suggestion
		const Point Lerp = [](const Point &a, const Point &b, const Point &c)
		{
			return a + (b - a) * c;
		};

		// Flip the velocity offset if cameraAccelMultiplier is negative to simplify logic.
		const Point absoluteOldCenterVelocity = Lerp(baseVelocity, oldCenterVelocity, cameraAccelMultiplier);

		const Point newAbsVelocity = Lerp(absoluteOldCenterVelocity, baseVelocity, CAMERA_VELOCITY_TRACKING);

		const Point newCenter = Lerp(oldCenter + newAbsVelocity, baseCenter, CAMERA_POSITION_CENTERING);

		// Flip the velocity back over the baseVelocity
		const Point newVelocity = Lerp(baseVelocity, newAbsVelocity, cameraAccelMultiplier);
```
always capitalize functions, also I feel we can just write Point here, no need for auto